### DISCUSSION TITLE
Downloading the minimum required

### DISCUSSION THREAD

**Discussion by arshavinel** [dompdf/dompdf#2402] [General]
I create small projects where I don't use composer at all and downloading Dompdf from here:

https://php-download.com/package/dompdf/dompdf

However, I got so many files Dompdf class don't use them at all (`.json`, `.md`, about composer, `.map`, about git, `.yml`, etc). I personally don't know which are used and which aren't.

There is any way downloading the minimum required files?
Only those used by Dompdf class when creating a PDF.

**Comment by bsweeney** [comment]
A self-contained packaged release can be found on the [releases page](https://github.com/dompdf/dompdf/releases).

We don't currently have a fully minimized packaged release (maybe we can start looking at doing that). Generally most projects only need files in either the dist or source directory, though Dompdf needs the files from the lib directory as well.

**Reply by arshavinel** [reply]
Hello. Thx. I will download it and let you know if is smaller than that from https://php-download.com

**Reply by arshavinel** [reply]
They have almost same number of files so doesn't actually help me.

Can't wait for a minimized packaged release in the future, if that will be possible. I personally play a lot with entire projects (merging them, moving with FTP, storing in clouds, saving in drivers), so less files and data really help.

Thx for your support :)

**Comment by arshavinel** [comment]
@bsweeney I added a new question very important for me. There is any way you could help me?

https://github.com/dompdf/dompdf/issues/2421
